packages/powersync/lib/src/database/web/web_powersync_database.dart

```diff
@@ -144,13 +145,14 @@ class PowerSyncDatabaseImpl
         updateStream: updates,
         retryDelay: Duration(seconds: 3),
         client: FetchClient(mode: RequestMode.cors),
+        syncParameters: params,
         // Only allows 1 sync implementation to run at a time per database
         // This should be global (across tabs) when using Navigator locks.
         identifier: database.openFactory.path);
     sync.statusStream.listen((event) {
       setStatus(event);
     });
-    sync.streamingSync(abortController: disconnecter);
+    sync.streamingSync();
```

Comment:
The `disconnecter` is used when the `disconnect` method is called, but now this is no longer tied to the `StreamingSyncImplementation`. I think we should be able to do (taken from Native implementation):

```Dart
 disconnector.onAbort.then((_) {
           sync.abort();
          }).ignore();
```

Reply:
Done! Thanks for the suggestion.